pkg/kubectl/cmd/util/factory.go

```diff
@@ -343,46 +343,28 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 		},
 		ClientForMapping: func(mapping *meta.RESTMapping) (resource.RESTClient, error) {
 			gvk := mapping.GroupVersionKind
-			mappingVersion := mapping.GroupVersionKind.GroupVersion()
-			c, err := clients.ClientForVersion(&mappingVersion)
-			if err != nil {
-				return nil, err
-			}
 			switch gvk.Group {
 			case api.GroupName:
+				mappingVersion := mapping.GroupVersionKind.GroupVersion()
```

Comment:
All that is different between this and default should be that APIPath is "/api" vs "/apis"